src/Templating/Helper/PhoneNumberHelper.php

```diff
@@ -31,18 +32,16 @@ public function __construct(PhoneNumberUtil $phoneNumberUtil)
         $this->phoneNumberUtil = $phoneNumberUtil;
     }
 
-    public function format(PhoneNumber|string $phoneNumber, string|int $format = PhoneNumberFormat::INTERNATIONAL): string
+    public function format(PhoneNumber|string $phoneNumber, int|PhoneNumberFormat $format = PhoneNumberFormat::INTERNATIONAL): string
```

Comment:
I think this is a major pain point for templating. I need to analyze more what we can do to have a nice API here.

Reply:
It's possible to deprecate the type `int` in the current version to simplify the API in the next major version.